examples/vercel-ai-chatbot/app/api/chat/route.ts

```diff
@@ -23,6 +22,19 @@ const requestBodySerializer = serialization.object({
   messages: serialization.list(ChatMessageSerializer)
 })
 
+class StreamingTextResponse extends Response {
```

Comment:
This is a copy paste of Vercel's minus some random experimental `data` arg they had as a third param